client/state/themes/test/selectors.js

```diff
@@ -66,6 +68,13 @@ const twentyfifteen = {
 	author_uri: 'https://wordpress.org/',
 };
 
+const freeThemeTier = {
+	slug: 'free',
+	name: 'Free',
+	plan: PLAN_FREE,
+	feature: FEATURE_WOOP,
```

Comment:
Why do we use `FEATURE_WOOP` and not its real feature (null)? 🤔

Reply:
It's a test, I would use ANY_FEATURE but we don't have one XD.